src/v2/Apps/Order/Routes/Offer/index.tsx

```diff
@@ -155,15 +155,22 @@ export class OfferRoute extends Component<OfferProps, OfferState> {
     const listPriceCents = this.props.order.totalListPriceCents
     const artworkPrice = artwork?.price
     const isInquiryCheckout = !artwork?.isPriceRange && !artwork?.price
-    const isEdition = artwork?.isEdition
+    const hasPrice =
+      (artwork?.listPrice?.__typename === "Money" &&
+        artwork?.listPrice?.major) ||
+      (artwork?.listPrice?.__typename === "PriceRange" &&
+        artwork?.listPrice?.maxPrice?.major &&
+        artwork?.listPrice?.minPrice?.major)
+    const showPriceOptions =
+      (artwork?.editionSets?.length ?? 0) < 2 && !!hasPrice
```

Comment:
This is just replacing `isEdition`, to see if we should be showing the "offer too low" alert on this particular flow.